### DISCUSSION TITLE
Trying to emulate our family 486 from 1991/92

### DISCUSSION THREAD

**Discussion by robman501a** [86Box/86Box#3132] [Q&A] [answered]
Hello, I would like help as I'm trying to emulate our first family computer so I can play the games which I still have. (Yeah, the CD's still work well)
I'm not sure how to configure 86Box for this. Could you please help me? My memory of this system is very weak as I was just a young kid still. 

Year: Approx 1991/1992
Motherboard: ??? It had ISA and VL-Bus
BIOS: AMI
CPU: 486DX2 66 (The case had a turbo button, it was 66 with the turbo on, 10 with the turbo off)
GPU: [VLB] Trident TGUI 9400CXi (I remember that BIOS logo at power up)
Sound: SoundBlaster 16 ISA bus (it might have been Vibra 16)
Floppy: 3 1/2 (drive A) and 5 1/4 (drive B)
HDD: Fujitsu IDE (Not sure on size)
CD-ROM: Not sure on the brand or speed but it was IDE
Modem: YES! Probably ISA bus.

At power up I remember this happening....

Trident BIOS
AMI BIOS POST screen

Floppy A and B seek

(booting from HDD)

Starting ONTRACK.......
(Blue square ONTRACK overlay logo)

Starting MS-DOS....

Autoexec would end with dosshell.

I remember seeing drives A and B as the floppies, C, D, and E as the HDD's, and F would be the CD-ROM

**Comment by TheVirtualComputer** [comment]
As far as I remember, some of the components listed (such as the Trident TGUI9400CXi) came later than 1992.

**Reply by robman501a** [reply]
Maybe it was one the 8900 series cards than and maybe it was ISA bus.

**Reply by robman501a** [reply]
It is possible that I was mistaken about the parts. If we had the system back in 1992 and never upgraded the parts, than I need to do some more researching.

**Comment by robman501a** [comment, marked as answer]
I figured it out.

CPU: 486 DX2 66
RAM 8MB
Motherboard: Chaintech 433SC
Display: Trident 8900 ISA
Input: Mouse Systems Serial Mouse
Network: N/A
Ports: LPT1 - Printer
	 Serial port 1 
	 Serial port 2
	 Parallel port 1

Storage Controller: HD - Internal
			  FD - Internal

Hard disks: IDE 0:0 C-967, H-16, S-31, Speed (1992) 3600 RPM (Can't recall the size so using 234MB)

Floppy drives: 3.5" 1.44M, 5.25" 360k


Added later after 1995.
Sound: SoundBlaster 16
CD-ROM: ATAPI 0:1, 4x
Swapped the smaller HDD to 4GB later on.
